src/test/java/seedu/address/model/transaction/DebtTest.java

```diff
@@ -22,6 +23,14 @@ public void equals() {
                 .withDescription("Supper")
                 .withAmount("10")
                 .withDate("2020-01-23").build();
+        Loan supperLoan = new LoanBuilder()
```

Comment:
hmmm i think this shouldn't be in DebtTest?

Reply:
this is to check if they are the different type of transactions